README.md

```diff
@@ -23,11 +23,34 @@ This repository hosts the primary codebase for the Minsta app.
 
 This repository hosts the primary codebase for the [Minsta App](https://minsta.me), originally created and developed by [@microchipgnu](https://github.com/microchipgnu).
 
-All new Minsta instances should be derived from this repository.  
+All new Minsta instances should be derived from this repository.
 
 [Video Tutorial on Setting up](https://www.loom.com/share/d5a038fb341c40be9ae131dd82f199a80)
 
 
+## Pre-Setup - Proxy Contract
+
+1. You got to have a NEAR Contract where you add a proxy contract as a minter.
+2. The proxy contract is meant for let any user who is not a minter, to be able to mint images on Minsta.
+3. The Near Contract will be where the NFT images will be minted
+4. The proxy contract will be the minter
+5. The user wallet address will be the owner of the NFT.
+
+### Deploying a Near Contract on Mintbase:
+1. Login on Mintbase and access [Contracts Page](https://www.mintbase.xyz/launchpad/contracts/0)
+2. Click on New Contract
+3. Choose Store Name (this will be the contract address to add on your minsta instance, this need to be added on the `process.env.NEXT_PUBLIC_NFT_CONTRACT_ADDRESS` env var.) and Store Symbol
```

Comment:
```suggestion
3. Choose Store Name (this will be the contract address to add on your minsta instance, this need to be added on the `process.env.NEXT_PUBLIC_NFT_CONTRACT_ADDRESS` environment variable) and Store Symbol
```